src/systems/media-frames.js

```diff
@@ -250,8 +250,13 @@ AFRAME.registerComponent("media-frame", {
 
   snapObject(capturedEl) {
     // TODO this assumes media frames are all in world space
```

Comment:
This PR fixes this todo so we should remove it.

Reply:
Updated.